main.go

```diff
@@ -761,7 +761,11 @@ func (c configImpl) HasCommandName(name string) bool {
 }
 
 func (c configImpl) SkipNeeds() bool {
-	return c.c.Bool("skip-needs")
+	if !c.IncludeNeeds() {
+		return c.c.Bool("skip-needs")
```

Comment:
@mumoshu I think it will return false. when --include-needs is not provided.

Reply:
Why? I `--include-needs` is `false` by default and in that case this falls into L765 and return `true`, as `BoolTFlag` defaults to `true`.

Reply:
oh. I know.